test/intl402/Temporal/PlainDate/from/chinese-calendar-dates.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2025 Google Inc, Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.from
+description: Test `from` with Chinese calendar
+features: [Temporal]
+---*/
+
+const testChineseData = new Date("2001-02-01T00:00Z").toLocaleString("en-US-u-ca-chinese", {
+  day: "numeric",
+  month: "numeric",
+  year: "numeric",
+  era: "short",
+  timeZone: "UTC"
+});
+const hasOutdatedChineseIcuData = !testChineseData.endsWith("2001");
```

Comment:
This is leftover from when these tests existed as part of the Temporal polyfill. When moving it out of staging, we should remove this and just assume engines have correct ICU data (or they will fail the test.)

Reply:
Fixed in [3f8f514](https://github.com/tc39/test262/pull/4667/commits/3f8f5149f22ae8bd5572ba26bf693d594da89af9)